TOOLS/odoo-tools.sh

```diff
@@ -1224,23 +1227,72 @@ if [ "$SCRIPT_MODE" = "updatetranslation" ]; then
 fi
 
 # ---------------------------------------------------------------------------------------
-# $ odoo-tools.sh backup      {TARGET_BRANCH} {SUPER_PASSWORD} {DBNAME}
+# $ odoo-tools.sh backup      {TARGET_BRANCH} {DBNAME} #  [TYPE]
 # ---------------------------------------------------------------------------------------
-MODEBACKUP="odoo-tools.sh backup {TARGET_BRANCH} {SUPER_PASSWORD} {DBNAME}"
+MODEBACKUP="odoo-tools.sh backup {TARGET_BRANCH} {DBNAME|all_running|all}" #  [TYPE]"
```

Comment:
To be discussed:
I think we should also add the possibility to backup all needed components of an instance so also ehterpad and owncloud Because in case of "ermergency" all of this component will be needed for a full odoo restore:

Therefore i think we need a version 2.0 of this backup script with a new parameter (we could use the planned parameter ```type``` already): ```[ odoozip | odoosql | etherpad | owncloud | full ]```

A ```full``` backup should also backup the configuration files of the odoo instance